portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,15 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+    const randomQuote =
```

Comment:
2-space indentation for JS too: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation